core/src/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -60,8 +60,8 @@ internal MsalAccessTokenCacheItem
             NormalizedScopes = scopes;
             TenantId = tenantId;
             Secret = secret;
-            ExpiresOnUnixTimestamp = CoreHelpers.DateTimeToUnixTimestamp(accessTokenExpiresOn);
-            CachedAt = CoreHelpers.CurrDateTimeInUnixTimestamp();
+            ExpiresOnUnixTimestamp = CoreHelpers.DateTimeToUnixTimestamp(accessTokenExpiresOn).ToString();
```

Comment:
Would it make sense to make the DateTimeToUnixTimestamp return as string or perhaps to add another helper e.g. DateTimeToUnixTimestampAsString ensuring people do not have to rememeber to add .ToString() ?